manifests/0000_90_kube-apiserver-operator_05_api_performance_dashboard.yaml

```diff
@@ -209,7 +209,7 @@ data:
           "timeFrom": null,
           "timeRegions": [],
           "timeShift": null,
-          "title": "etcd Request Duration - 99th Quantile",
+          "title": "etcd Request Duration - 99th Percentile",
```

Comment:
part of first commit?

Reply:
@sttts this is part of the second commit - it's either 0.99 quantile or 99th percentile.